crates/subspace-runtime/tests/integration/main.rs

```diff
@@ -0,0 +1 @@
+mod object_mapping;
```

Comment:
Hmm, why not just create a new module `object_mapping` or something like that under `subspace-runtime` and move the tests as a private module?

Reply:
This is made external intentionally because we are testing public API here, not implementation details